.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
```

Comment:
Hmmmm... Not sure where did I get that `setuptools` idea.

Anyway,

`RuntimeError: You are linking against OpenSSL 0.9.8, which is no longer support by the OpenSSL project. You need to upgrade to a newer version of OpenSSL.`. Here's the full Travis log. https://api.travis-ci.org/jobs/141770652/log.txt?deansi=true